app/notebook-store.js

```diff
@@ -92,6 +93,11 @@ function _getFilename (req) {
     return name;
 }
 
+function _hiddenFileFilter (filename) {
+    //check for hidden files or directories, returns true if file is not hidden
+    return !(/(^|\/)\.[^\/\.]/g).test(filename);
```

Comment:
Since we are simply doing a `readdir` to get a list of directory contents, can we just check for leading dot?

```javascript
return /^[^.]/.test(filename)
```